### PR TITLE
Db buildings

### PR DESCRIPTION
**Pull request by OleksandrZhydyk**

### CODE REVIEW

File: src/cart/models.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth import get_user_model
+from django.core.validators import MinValueValidator
+from django.db import models
+
+
+class Cart(models.Model):
+    user = models.ForeignKey(to=get_user_model(), on_delete=models.CASCADE)
+    creation_date = models.DateTimeField(auto_now_add=True)
```
Comment (mikolaz27) [inline src/cart/models.py:13]: if its type, better to use chooice

File: src/cart/models.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    payment_type = models.CharField(max_length=100, null=True)
+    payment_id = models.CharField(max_length=100, null=True)
+    ordered = models.BooleanField(default=False)
+
+
+class CartItem(models.Model):
+    part = models.ForeignKey("catalogue.Part", on_delete=models.CASCADE)
+    quantity = models.IntegerField(
```
Comment (mikolaz27) [inline src/cart/models.py]: positive

File: src/catalogue/models.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    discount_price = models.FloatField(verbose_name="Discount", blank=True, null=True)
+    date_created = models.DateTimeField(auto_now_add=True, null=True, editable=False, verbose_name="Part adding date")
+    image = models.ImageField(
+        default="profiles_avatars/empty_avatar.png",
+        null=True,
+        blank=True,
+        upload_to="profiles_avatars/%Y/%m/%d/",
+        verbose_name="Image",
+    )
+    remark = models.CharField(max_length=255, blank=True, null=True, verbose_name="Remark")
```
Comment (mikolaz27) [inline src/catalogue/models.py:24]: what is remark? maybe comments

Comment (OleksandrZhydyk) [reply src/catalogue/models.py:24]: Remark it is like additional info about part

File: src/cart/models.py (hunk 2 above)
Comment (OleksandrZhydyk) [reply src/cart/models.py]: Ok, I will change

File: src/cart/models.py (hunk 1 above)
Comment (OleksandrZhydyk) [reply src/cart/models.py:13]: I'm not clearly understand about choice field, do you mean some exact field to use the choice?
I'm also not sure of how to implement the customer's shop basket in database level.

File: src/cart/models.py (hunk 2 above)
Comment (OleksandrZhydyk) [reply src/cart/models.py]: After change, we can delete the validator, doesn't it?

**Pull request merged**